src/core/runtime.d

```diff
@@ -313,13 +313,54 @@ struct Runtime
     }
 
 
+    /**
+     * Set source file path for coverage reports.
+     *
+     * Params:
+     *  path = The new path name.
+     */
+    static void coverSourcePath(string path)
```

Comment:
Documenting the functionality.
Some extern C functions buried in the runtime don't qualify as API.